src/components/signup/style.module.css

```diff
@@ -0,0 +1,52 @@
+.container{
+  text-align: center;
+}
+.title{
+    font-size: 2rem;
+}
+.subline{
+    margin: 1rem 0;
+    font-size: 1rem;
+}
+.userDetails{
+    font-size: 0.7rem;
+    text-align: left;
+    margin: 0 2.75rem;
+}
+.label{
+    display: inline-block;
+    margin: 0.48rem;
+    width: 6rem;
+}
+.input{
+    width: 8rem;
+    border-bottom: 0.1rem solid var(--modal-border);
+}
+
+
+@media (min-width: 600px){
```

Comment:
Check if we can use css variabe in min-width. Check first and if it is feasible create a var in dimens.css